Formula/p/percona-server.rb

```diff
@@ -63,7 +63,7 @@ class PerconaServer < Formula
 
   # https://github.com/percona/percona-server/blob/Percona-Server-#{version}/cmake/boost.cmake
   resource "boost" do
-    url "https://boostorg.jfrog.io/artifactory/main/release/1.77.0/source/boost_1_77_0.tar.bz2"
```

Comment:
jfrog link is dead, switch to use the one from sourceforge

<img width="1051" alt="image" src="https://github.com/user-attachments/assets/ced8aa15-7934-449e-9953-09114ca125c8" />